android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -118,6 +124,8 @@ public class BravePrivacySettings extends PrivacySettings implements ConnectionE
             PREF_BRAVE_SOCIAL_BLOCKING_SECTION, // social blocking section
             PREF_SOCIAL_BLOCKING_GOOGLE, PREF_SOCIAL_BLOCKING_FACEBOOK,
             PREF_SOCIAL_BLOCKING_TWITTER, PREF_SOCIAL_BLOCKING_LINKEDIN,
+            PREF_YOUTUBE_SECTION, // Youtube section
+            PREF_HIDE_YOUTUBE_RECOMMENDED_CONTENT, PREF_HIDE_YOUTUBE_DISTRACTING_ELEMENTS,
```

Comment:
need for check format for `PREF_HIDE_YOUTUBE_DISTRACTING_ELEMENTS`

Reply:
Not getting format specific error.
npm run format
works fine